samples/nrf9160/modem_shell/README.rst

```diff
@@ -16,6 +16,8 @@ The sample supports the following development kits:
 
 .. table-from-sample-yaml::
 
+.. include:: /includes/tfm.txt
```

Comment:
I just created PR #7743 to switch `modem_shell` back to SPM. With TF-M there's not enough RAM left for the application in all cases.

Reply:
reverted to spm.txt